src/commands/build/watch/mod.rs

```diff
@@ -34,7 +38,7 @@ pub fn watch_and_build(
                 message::info(&format!("watching {:?}", &JAVASCRIPT_PATH));
 
                 loop {
-                    match wait_for_changes(&watcher_rx, COOLDOWN_PERIOD) {
+                    match wait_for_changes(&watcher_rx, COOLDOWN_PERIOD, None) {
```

Comment:
what does the third argument in this wait_for_changes function do? looks new?

Reply:
This argument provides a set of directories whose contents live reload should ignore. So far only Rust Workers have explicit ignored directories.

Reply:
removed third argument with latest version :)